tests/test_api_v1.py

```diff
@@ -126,16 +131,26 @@ def test_fetch_paginated_requests(mock_chain, app, auth_env, client, db):
     assert len(fetched_requests) == 20
     for repo_number, request in enumerate(fetched_requests):
         assert request['repo'] == repo_template.format(repo_number)
+    assert response['meta']['previous'] is None
+    assert fetched_requests[0]['dependencies'] == 13
 
     # per_page and page parameters are honored
-    rv = client.get('/api/v1/requests?page=2&per_page=10')
+    rv = client.get('/api/v1/requests?page=2&per_page=10&verbose=True')
     assert rv.status_code == 200
     response = json.loads(rv.data.decode('utf-8'))
     fetched_requests = response['items']
     assert len(fetched_requests) == 10
     # Start at 10 because each page contains 10 items and we're processing the second page
     for repo_number, request in enumerate(fetched_requests, 10):
         assert request['repo'] == repo_template.format(repo_number)
+    pagination_metadata = response['meta']
+    assert pagination_metadata['next'].endswith('?page=3&per_page=10')
```

Comment:
I'll try to clarify my previous comment. My concern is that `?page=3&per_page=10` will not always be in the same order, so the tests will pass on some systems and not on others. To be clear, I think the URL could end up being `?page=3&per_page=10` or `?per_page=10&page=3`.